Testing/Tools/cxxtest/python/cxxtest/__init__.py

```diff
@@ -2,7 +2,7 @@
 
 .. _CxxTest: http://cxxtest.tigris.org/
 
-The _CxxTest testing framework is focussed on being a lightweight
+The _CxxTest testing framework is focused on being a lightweight
```

Comment:
Isn't **focussed** on the whitelist :confused:

Reply:
Yea, its PEBKAC. I `git stash`ed some commits before I submitted the previous PRs and I keep `git stash pop`ing those commits and appending tthem to these current PRs when they are obsolete. So I need to just go back and clean out any obsolete issues like this one. Apologies.

Reply:
fixed in 10d2a39